index.html

```diff
@@ -6,81 +6,80 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet">
-    <link rel="stylesheet" href="index.css">
+      
+ 
+
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.1.1/css/fontawesome.min.css" class="rel">
     <title>Mobile Version Vkeleton</title>
 </head>
 <body>
-
     <main>
-       
-            <header>
+            <div class="navbar">
             <nav>
-                <div class="image">
-                <img src="images.jpeng/Light.png" alt="">
-                </div>
-                <div class="nav-wrapp">
+                    <a href="#"><h3>My Logo</h3></a>
+             
+                <div class="right-nav" id="rightNav">
+                    <ul class="nav-links">
+
+                        <li id="close" class="material-icons" onclick="hideMenu()">close</li>
+
+                        <li><a class="navlink" onclick="hideMenu()" href="#portfolio">Portfolio</a></li>
+
+                        <li><a class="navlink" onclick="hideMenu()" href="#About">About</a></li>
+
+                        <li><a class="navlink" onclick="hideMenu()" href="#Contact">Contact</a></li>
```

Comment:
- [ ] Good job working on this, but there is whitespace around the mobile menu which does not create a good user experience. Please remove the whitespace to the left and to the right of the menu.

![image](https://user-images.githubusercontent.com/43995830/168775786-ceef8baf-db40-4aaa-8aa4-cf21e50952b0.png)

![image](https://user-images.githubusercontent.com/43995830/168775875-4a662e43-fc73-4384-801e-a6159b67ed4b.png)